### ISSUE TITLE
naming discrepancy causes problem in routes

### ISSUE THREAD

**Issue by lucas-nelson** [efexen/messenger_platform#23]
This is in my Gemfile:

```
gem 'messenger_platform_rails', '~> 0.5.0'
```

When I added:

```
mount MessengerPlatform::Engine, at: '/fb_webhook'
```

to my `routes.rb`, I saw this error on startup:

```
.../config/routes.rb:131:in `block in <top (required)>': uninitialized constant MessengerPlatform (NameError)
    from .../.rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/routing/route_set.rb:432:in `instance_exec'
    from.../.rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/routing/route_set.rb:432:in `eval_block'
    from .../.rvm/gems/ruby-2.3.1/gems/actionpack-4.2.6/lib/action_dispatch/routing/route_set.rb:410:in `draw'
    from .../config/routes.rb:1:in `<top (required)>'
```

I was able to fix by putting this at the top of my `routes.rb`:

```
require 'messenger_platform'
```

While rails now starts without error it felt a bit odd. Did some reading and found this: http://edgeguides.rubyonrails.org/engines.html#mounting-the-engine

> As described earlier, by placing the gem in the Gemfile it will be loaded when Rails is loaded. It will first require lib/blorgh.rb from the engine, then lib/blorgh/engine.rb, which is the file that defines the major pieces of functionality for the engine.

I suspect the root cause is a naming discrepancy between your repo and what's in rubygems.
Rubygems: messenger_platform_rails
repo: messenger_platform

In this case, there is no lib/messenger_platform_rails.rb or lib/messenger_platform_rails/engine.rb available (what's there lacks the '_rails' suffix.

Is there any way the names an be brought inline?

**Comment by efexen** [comment]
Hey @LucasAU,

Thanks for sharing this with me, the '_rails' bit was done in a rush as I took too long from starting to write the gem and someone else pinched the name ¯\_(ツ)_/¯

I didn't want to change the interface so I just added a `messenger_platform_rails.rb` that requires the messenger_platform and it resolves this issue.

I've released a new version of the gem v0.5.1 with that fix in as well as your PR #24 which was much appreciated!

Are you ok for me to close this issue?

Thanks,
.FxN
